kernel/CMakeLists.txt

```diff
@@ -94,6 +94,16 @@ list(APPEND kernel_files
 endif()
 
 add_library(kernel ${kernel_files})
+target_link_libraries(kernel c_picolibc)
+target_link_libraries(kernel c_common)
+
+add_library(c INTERFACE IMPORTED GLOBAL)
+add_library(gcc INTERFACE IMPORTED GLOBAL)
```

Comment:
This probably needs to be set by the toolchain; we could switch to the llvm equivalent at some point?